dlp/snippets/deid.py

```diff
@@ -1938,6 +1938,195 @@ def deidentify_table_with_multiple_crypto_hash(
 # [END dlp_deidentify_table_with_multiple_crypto_hash]
 
 
+# [START dlp_deidentify_table_fpe]
+import base64  # noqa: F811, E402, I100
+from typing import Dict, List, Union  # noqa: F811, E402, I100
+
+import google.cloud.dlp  # noqa: F811, E402
+
+
+def deidentify_table_with_fpe(
+    project: str,
+    table_data: Dict[str, Union[List[str], List[List[str]]]],
+    deid_content_list: List[str],
```

Comment:
Can this be named `deid_field_names`? I think this is a list of field names, but with "content" it's confusing and could be interpreted as infotypes instead of field names.

Reply:
(same for `reid_` below)